packages/sales-api-service/src/services/permit.service.js

```diff
@@ -0,0 +1,41 @@
+import { getReferenceDataForEntity } from './reference-data.service.js'
+import { Permit, Concession, PermitConcession } from '@defra-fish/dynamics-lib'
+
+export const findPermit = async existingPermission => {
+  const licenseeConcessions = existingPermission.concessions || []
+  const permitsJoinPermitConcessions = await getPermitsJoinPermitConcessions()
+
+  if (licenseeConcessions.some(c => !c.name && !!c.type)) {
+    throw new Error("Concession should contain 'name' not 'type'")
```

Comment:
What's the reason for checking for this? Do we have a data quality issue we need to sort out?

Reply:
I put it in because I've seen a lot of usage of 'type' rather than 'name', a hold-over (I think) of copy-and-paste from the GAFL easy renewals. The recurring payments code now uses name rather than type, but I thought I'd put in an error as if type is specified in concessions rather than name, the problem is not at all obvious and may well trip us up when applying to Easy Renewals.